src/hooks/context-menu-fixed-style/index.ts

```diff
@@ -82,12 +79,12 @@ export const useContextMenuFixedStyle = ({ useFixedMenuStyle, visibleMenu }: Sta
         contextMenuFixedStyleState.contextMenuStyle = contextMenuStyle;
     };
 
-    watch(() => contextMenuFixedStyleState.proxyVisibleMenu, async () => {
-        if (!contextMenuFixedStyleState.proxyVisibleMenu || !contextMenuFixedStyleState.targetRef) {
+    watch(() => visibleMenu.value, async () => {
+        if (!visibleMenu.value || !contextMenuFixedStyleState.targetRef) {
```

Comment:
```suggestion
    watch(() => state.visibleMenu, async (visible) => {
        if (!visible || !contextMenuFixedStyleState.targetRef) {
```

what about this ? :)

Also, you can use like this, with ref:

```suggestion
    watch(visibleMenu, async (visible) => {
        if (!visible || !contextMenuFixedStyleState.targetRef) {
```

Reply:
Oh.. I missed staging that changes. Thank you for catching. 
I just realize another way you suggest.